### PR TITLE
[tests-only] Reword Spaces API scenario descriptions

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
These are suggested scenario descriptions for the `apiSpaces` features.

Discussion is welcome! I think we should remove the detail about expected HTTP status and other expectations from the description. Those are written in the test steps. The description can just be a "short" sentence about what the scenrio is supposed to test.

Some scenarios were already in the form "some user can do something", so I kept with that "can do"/"cannot do" form. It reads nicely for me. I removed specific mention of Alice and Brian from the descriptions - that is just test step detail that is intended to help someone who reads the scenario.

What do you think?

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpaces/quota.feature [hunk 1]
```diff
@@ -37,32 +37,31 @@ Feature: State of the quota
...
-    Given user "Alice" has created a space "Project Alfa" of type "project" with quota "10" 
+
+  Scenario: A file cannot be uploaded if there is insufficient quota
+    Given user "Alice" has created a space "Project Alfa" of type "project" with quota "10"
     When user "Alice" uploads a file inside space "Project Alfa" with content "More than 10 bytes" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "507"
 
 
-  Scenario: creating a folder with an insufficient quota
+  Scenario: A folder cannot be uploaded if there is insufficient quota
```
Comment (individual-it) [inline tests/acceptance/features/apiSpaces/quota.feature:47]:
```suggestion
  Scenario: A folder cannot be created if there is insufficient quota
```

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/features/apiSpaces/changeSpaces.feature [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @api @skipOnOcV10
 Feature: Change data of space
-  As a user
-  I want to be able to change the data of the created space(increase the quota, change name, etc.)
+  As an admin user
```
Comment (ScharfViktor) [inline tests/acceptance/features/apiSpaces/changeSpaces.feature:3]: As a user with "admin rights". There will be cases with other roles later

File: tests/acceptance/features/apiSpaces/quota.feature (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/apiSpaces/quota.feature:47]:
```suggestion
  Scenario: A folder can be created even if there is insufficient quota for file content
```

Comment (phil-davis) [reply tests/acceptance/features/apiSpaces/quota.feature]: @individual-it thanks for spotting. I looked more closely and a folder can be created in this case. Folders do not consume quota. So I changed even more words.

**Review by ScharfViktor — APPROVED** [review]

File: tests/acceptance/features/apiSpaces/changeSpaces.feature (hunk 2 above)
Comment (phil-davis) [reply tests/acceptance/features/apiSpaces/changeSpaces.feature]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**